app/controllers/session.rb

```diff
@@ -0,0 +1,20 @@
+get '/session/new' do
+  erb :'/session/login'
+end
+
+
+post '/session' do
+  user = User.authenticate(params[:user])
+  if user
+    session['user_id'] = user.id
+    redirect '/suggestions'
+  else
+    @error = "Error"
+    erb :'/session/new'
+  end
+end
+
+get '/session/delete' do
```

Comment:
Looks like you've got two different routes for logout. Let's keep one of these, and remove the other, to maintain the DRY principle (do not repeat yourself).